app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,52 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+h1,h2,h3,h4 {
+	color: #24C73A;
+}
+
+body {
+	background-color: #fedf6b;
+}
+
+.container-fluid {
+	background-image: url('adventure.jpg');
+	margin: 66px -129px 7px 10px;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered background-image, background-size, margin